applications/lib_app/rsa_util/rsa_util.cpp

```diff
@@ -252,3 +252,49 @@ bool openssl_gen_dh(const char* dh_key, int bitsize)
     return true;
 #endif
 }
+
+bool get_server_subject(const char *server_crt_path, char *server_subject, size_t size_name)
```

Comment:
We should merge 2 methods to get_subject_crt(const char *crt_path, char *subject, size_t size_name)